apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -364,10 +365,32 @@ const Ticket = new GQLListSchema('Ticket', {
         lastCommentAt: {
             schemaDoc: 'Last comment time in ticket',
             type: DateTimeUtc,
+            access: {
+                create: userIsAdmin,
+                read: true,
+                update: userIsAdmin,
+                delete: false,
```

Comment:
Why `userIsAdmin`?
It should be `canOnlyServerSideWithoutUserRequest` or `() => false`